perses/app/relative_point_mutation_setup.py

```diff
@@ -116,8 +116,8 @@ def __init__(self,
                 if phase == vacuum, then the complex will not be solvated with water; else, it will be solvated with tip3p
             conduct_endstate_validation : bool, default True
                 whether to conduct an endstate validation of the hybrid topology factory
-            ligand_file : str, default None
-                path to ligand of interest (i.e. small molecule or protein); .sdf or .pdb
+            ligand_input : str or list of oemol objects, default None
```

Comment:
@glass-w  `list of oemol objects` should be changed to `oemol` here

Reply:
Ah yes good catch!